projects/plugins/boost/app/admin/class-config.php

```diff
@@ -183,11 +182,13 @@ public static function set_getting_started( $value ) {
 	 */
 	public static function is_getting_started() {
 		// Aside from the boolean flag in the database, we also assume site already got started if they have premium features.
-		return \get_option( 'jb_get_started', false ) && ! Premium_Features::has_feature( Premium_Features::CLOUD_CSS ) && ! ( new Status() )->is_offline_mode();
```

Comment:
With the removal of `jb_get_started`, I'm thinking that we don't need logic tied to it at all. Should we add a ticket to clean this up in a separate PR?